scripts/ci/github/check_labels.sh

```diff
@@ -46,6 +52,16 @@ else
   exit 1
 fi
 
+if has_label  "$repo" "$pr" 'B1-note_worthy'; then
```

Comment:
It would be better to check for the code only: `B1` rather than the label string: `B1-note_worthy`.
Here everything will collapse if we ever rename `B1-note_worthy` to something like `B1-Interesting ℹ️ ` (for whatever other text).

I would suggest adding a function such as:
```
get_code() {
    [[ $1 =~ ^([A-Z][0-9]).* ]] && echo "${BASH_REMATCH[1]}"
}
```
to extract the label "code" and compare only those label codes.

Reply:
It may be also convenient to handle a list of tuples with the code only and the full string:
```
for i in B1,B1-note_worthy X1,X1-whatever; do
    IFS=","
    set -- $i
    echo $1 and $2;
done
```

Reply:
I guess adding the `get_code()` function requires some refactor. IMO it's not a good option to make this bash-script (or `lib.sh`) more complicated. At this moment I would suggest merging it as is and create an issue for the OpsTooling team to make a proper service for label checks for all 3 repos (cumulus, polkadot and substrate)

Reply:
ok, that works too